owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.java

```diff
@@ -309,10 +303,9 @@ private void openFeedback() {
         String feedbackMail = (String) getText(R.string.mail_feedback);
         String feedback = "Android v" + BuildConfig.VERSION_NAME + " - " + getText(R.string.drawer_feedback);
         Intent intent = new Intent(Intent.ACTION_SENDTO);
-        intent.setType("text/plain");
         intent.putExtra(Intent.EXTRA_SUBJECT, feedback);
 
-        intent.setData(Uri.parse(feedbackMail));
+        intent.setDataAndType(Uri.parse(feedbackMail), "text/plain");
```

Comment:
I think that it could be an issue itself. Moving just this string to a constant looks weird since there are files like MimetypeIconUtil where every mime is hardcoded, and we will mix hardcoded and constants. I think it is better to standarize and move everything together, don't you think?

Reply:
Ok, could you create with the details? Thanks